examples/montage/packages/todomvc-common@13305a5/readme.md

```diff
@@ -0,0 +1,15 @@
+# todomvc-common
+
+> Common TodoMVC utilities used by our apps
+
+
+## Install
+
+```
+$ npm install --save todomvc-common
+```
+
+
+## License
+
+MIT © [TasteJS](http://tastejs.com)
```

Comment:
We only want the vendor files which we need to be committed. [We ignore the rest.](https://github.com/marchant/todomvc/blob/master/examples/mithril/.gitignore)